pkg/combustion/templates/15-k3s-single-node-installer.sh.tpl

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -euo pipefail
+
+mount /var
+mkdir -p /var/lib/rancher/k3s/agent/images/
+cp {{ .imagesPath }}/* /var/lib/rancher/k3s/agent/images/
```

Comment:
I'll definitely revisit this once the full k3s implementation (both single and multi node) is done. I'm not sure how much of it will be reusable between k3s and RKE2 but it's definitely something I have in mind.